0-bootstrap/README-Jenkins.md

```diff
@@ -159,7 +180,7 @@ You arrived to these instructions because you are using the `jenkins_bootstrap`
     sed -i "s/UPDATE_ME/$TF_STATE_GCS_BUCKET_NAME/" backend.tf
     ```
 
-   **If using MacOS:**
+   **If using MacOS:** sed command works different
```

Comment:
I am still a bit confused if we need two sets of instructions. The `sed -i ".bak"` works across both but again we are now providing instructions per platform. if we are agreeing on providing instructions for both FreeBSD and GNU sed then maybe we should standardize this across the whole repo?
@daniel-cit @rjerrems wdyt?

Reply:
@daniel-cit alerted me about the possible issue of people wanting instructions per platform I added this. I would like to hear from the rest of you to see it makes sense to remove these steps altogether.

Reply:
@caleonardo @bharathkkb I would replace from `Run this command to copy the backend.tf` to `rm backend.tf.bak`

with just:

1. Rename backend.tf.example to backend.tf
1. Run `terraform output gcs_bucket_tfstate` to get the tfstate bucket name
1. Edit file `backend.tf` and replace `UPDATE_ME` with the tfstate bucket name

Reply:
Done. Thank you.